meetup/models.py

```diff
@@ -1,29 +1,24 @@
 from django.db import models
 from pizzaplace.models import PizzaPlace
 from django.core.validators import RegexValidator
-from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+from meetup.services.meetup_api import MeetupApi
 from django.core.exceptions import ValidationError
-
+from model_utils.models import TimeStampedModel
 
 def validate_urlname(link):
   validator = RegexValidator(
     regex='meetup\.com\/\w+(-\w+)*\/$',
```

Comment:
Here's a helpful tool for getting Python regex right: http://pythex.org/. (That doesn't mean you don't need unit tests, too). 

I'm not sure whether you really need the capture group `(-\w+)` in the above. Are you using parentheses to try to group regex symbols? If so, take another look at the syntax.